File: sunbin/day4/python/001_삼각형의완성조건1.py
# https://school.programmers.co.kr/learn/courses/30/lessons/120889

def solution(sides):
    max_side = max(sides)
    
    sides.remove(max_side)
    other_sides_sum = sum(sides)
    
    
    if max_side < other_sides_sum:
        return 1
    else:
        return 2

# 다른 풀이
def solution(sides):
    return 1 if max(sides) < (sum(sides) - max(sides)) else 2

File: sunbin/Lv0/day4/001_삼각형의완성조건1.js
// https://school.programmers.co.kr/learn/courses/30/lessons/120889

function solution(sides) {
    sides = sides.sort((a,b) => a-b)
    return sides[0]+sides[1] > sides[2] ? 1 : 2;
}

solution([1, 2, 3])
solution([199, 72, 222])
